### ISSUE TITLE
PhoneFormatter is not working if Phone Number has +... prefix

### ISSUE THREAD

**Issue by akhripko** [BroilerBear/test-task#13]
try handle phone number like +38...

**Comment by BroilerBear** [comment]
Agree, this func should handle more wide cases for different prefix and formats.
Current implementation was done assuming that all numbers are valid UK ones (as described in requirements)
Will implement enhancement and test till tomorrow noon.
